### PR TITLE
Add python scripts to show and delete MPU

### PR DESCRIPTION
**Pull request by justinknierim**
We have customers on occasion who see large disk usage for unknown reasons, and it is sometimes old MPU data.  Add two scripts to print out the MPU data for a bucket and purge it.

Also "BUCKET_NAME" and "BUCKET NAME" were both used in scripts, standardized to "BUCKET NAME" to keep it like the other "ACCESS KEY" and "SECRET KEY" variables.

Also the tags seemed unrelated to this article, updated them to what I think would be good.

### CODE REVIEW

File: source/dreamobjects/tutorials/how-to-use-python-boto-scripts-to-manage-dreamobjects.rst [hunk 1]
```diff
@@ -144,4 +211,4 @@ For more examples of Python code using DreamObjects, please see
 .. _Python: 215489338-Installing-virtualenv-and-custom-modules-in-Python
 
 .. meta::
-    :labels: linux mac windows aws awscli
+    :labels: python boto scripts dreamobjects
```
Comment (smaffulli) [inline source/dreamobjects/tutorials/how-to-use-python-boto-scripts-to-manage-dreamobjects.rst]: Why remove linux mac windows? the idea of those labels is that (maybe some day) provide navigations for articles based on the OS. Also, dreamobjects is redudant as a label.

Comment (justinknierim) [inline source/dreamobjects/tutorials/how-to-use-python-boto-scripts-to-manage-dreamobjects.rst]: Interesting, I've never heard of the labels being used for it.  Technically anything can run python so I can add those 3 back.  The "aws awscli" has nothing at all to do with this article.

Comment (justinknierim) [inline source/dreamobjects/tutorials/how-to-use-python-boto-scripts-to-manage-dreamobjects.rst]: If you would prefer it split into another article, that can be done.  It didn't seem big enough to warrant that, but if you prefer please confirm and I'll do that!

Comment (smaffulli) [inline source/dreamobjects/tutorials/how-to-use-python-boto-scripts-to-manage-dreamobjects.rst]: Let's split it: it's better for many reasons (and we avoid getting back into the huge walls of texts we had on previous versions of the wiki)

**Comment by justinknierim** [comment]
I think the new page looks pretty good, and is long enough to warrant its own page.  Hope it looks better now, thanks!

**Comment by justinknierim** [comment]
Thanks Caleb!  Fixed up those future tense issues I believe.

**Comment by squidboylan** [comment]
:+1: from me

**Pull request merged**